privacy-policy.ftd

```diff
@@ -4,6 +4,7 @@
 
 
 -- lib.page:
+active-page: privacy-policy/
```

Comment:
@priyanka9634  as mentioned above comment - plz try to avoid additional properties to `page` component. Try a different approach.